cdc/http_handler.go

```diff
@@ -28,6 +28,10 @@ const (
 	APIOpVarAdminJob = "admin-job"
 	// APIOpVarChangefeedID is the key of changefeed ID in HTTP API
 	APIOpVarChangefeedID = "cf-id"
+	// APIOpVarTargetCaptureID is the key of to-capture ID in HTTP API
+	APIOpVarTargetCaptureID = "target-cp-id"
```

Comment:
Please do not abbreviate terms in API.
```suggestion
	APIOpVarTargetCaptureID = "target-capture-id"
```

Reply:
https://github.com/pingcap/ticdc/pull/593/files#diff-20d40ae92eaf96d272e76a7985938f91R30
Should we be unified with him `cf-id`?

Reply:
Oh yes please!